routers/private/repository.go

```diff
@@ -56,15 +56,15 @@ func GetRepository(ctx *macaron.Context) {
 func GetActivePullRequest(ctx *macaron.Context) {
 	baseRepoID := ctx.QueryInt64("baseRepoID")
 	headRepoID := ctx.QueryInt64("headRepoID")
-	baseBranch, err := url.QueryUnescape(ctx.QueryTrim("baseBranch"))
+	baseBranch, err := url.PathUnescape(ctx.QueryTrim("baseBranch"))
 	if err != nil {
 		ctx.JSON(http.StatusInternalServerError, map[string]interface{}{
 			"err": err.Error(),
 		})
 		return
 	}
 
-	headBranch, err := url.QueryUnescape(ctx.QueryTrim("headBranch"))
+	headBranch, err := url.PathUnescape(ctx.QueryTrim("headBranch"))
```

Comment:
`ctx.QueryTrim` will already unescape its contents so you don't actually need to unescape these.

Reply:
same as above